tools/seedDatabase.js

```diff
@@ -74,9 +74,777 @@ import { url } from "../server/config";
+          rightCardDefaultSliderPosition: 100,
+          marks: [
+            { position: 0, label: "Teacher-managed" },
+            { position: 100, label: "Student-managed" },
+          ],
+        },
+        {
+          chartID: chart3id,
+          score: 29,
+          note:
+            "Parallel parking video games playing my guitar snowboarding. Mountain biking Family Guy local sports teams exploring the city local sports teams, not",
+          definition: "Learning process",
+          type: "Practice",
+          leftCardStatement:
+            "Students reproduce accepted discipline knowledge.",
+          rightCardStatement:
+            "Students learn to construct their own understanding of the discipline.",
+          leftCardDefaultSliderPosition: 0,
+          rightCardDefaultSliderPosition: 100,
+          marks: [
+            { position: 0, label: "Reproduction" },
+            { position: 100, label: "Construction" },
+          ],
+        },
+        {
+          chartID: chart3id,
+          score: 18,
+          note:
+            "Parallel parking video games playing my guitar snowboarding. Mountain biking Family Guy local sports teams exploring the city local sports teams, not",
+          definition: "Feedback",
+          type: "Practice",
+          leftCardStatement: "Left card statement", // TODO: TBC
+          rightCardStatement: "Right card statement", // TODO: TBC
+          leftCardDefaultSliderPosition: 0, // TODO: TBC
+          rightCardDefaultSliderPosition: 100, // TODO: TBC
+          marks: [
+            { position: 0, label: "Minimal" },
+            { position: 50, label: "Fixed" },
+            { position: 100, label: "Responsive" },
+          ],
+        },
+        {
+          chartID: chart3id,
+          score: 18,
+          note:
+            "Parallel parking video games playing my guitar snowboarding. Mountain biking Family Guy local sports teams exploring the city local sports teams, not",
+          definition: "Learning goals (Curriculum beliefs)",
+          type: "Beliefs",
+          leftCardStatement:
+            "Learning goals are clearly defined and can be linked to assessment clearly.",
+          rightCardStatement:
+            "Learning goals are open to interpretation and are not easily assessable.",
+          leftCardDefaultSliderPosition: 0,
+          rightCardDefaultSliderPosition: 100,
+          marks: [
+            { position: 0, label: "Sharply-focused" },
+            { position: 100, label: "Unfocused" },
+          ],
+        },
+        {
+          chartID: chart3id,
+          score: 18,
+          note:
+            "Parallel parking video games playing my guitar snowboarding. Mountain biking Family Guy local sports teams exploring the city local sports teams, not",
+          definition: "Student collaboration (Curriculum beliefs)",
+          type: "Beliefs",
+          leftCardStatement: "Left card statement", // TODO: TBC
```

Comment:
These todo's?

Reply:
They should be changed later so they can stay.